packages/test/test-gc-sweep-tests/src/test/gcSweepTest.spec.ts

```diff
@@ -119,196 +84,153 @@ describeNoCompat("GC Random tests", (getTestObjectProvider) => {
 
     let overrideLogger: IgnoreErrorLogger;
 
-    let mainContainer: IContainer;
-    let connectedContainers: IContainer[];
-    let closedContainers: IContainer[];
-    let testNodes: Map<string, TestNode>;
+    const sleep = async (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
+    const seed = 0.0604202162258638; // Math.random();
+    const random: IRandom = makeRandom(seed);
```

Comment:
I see this is used throughout.  I'm skeptical whether it will give consistent test output, if things run in parallel at all. What do you think?

Reply:
I'll need to do more work to extract out the randomization logic and make the test repeatable, I see I need to be reusing `Math.random()`

Reply:
Created a task AB#1587